show deleted records CHM-665

tahmeedshah opened the pull request:
### General description
testing links:
http://localhost:2010/en/register/CNA/78AFAA88-47EA-9333-AD6E-C21BCB17DE21/view
http://localhost:2010/en/database/CON/BCH-CON-SCBD-257271/2

### Designs
_Link to the related design prototypes (if applicable)_

### Testing instructions
_Provide minimal instructions on how to test this PR._

* Apart from the added feature / bug fix, check overall performance, styling...

## Checklist before merging
* [ ]  Branch name / PR includes the related Jira ticket Id.
* [ ]  Tests to check core implementation / bug fix added.
* [ ]  All checks in Continuous Integration workflow pass.
* [ ]  Feature functionally tested by reviewer(s).
* [ ]  Code reviewed by reviewer(s).
* [ ]  Documentation updated (README, CHANGELOG...) (if required)

blaisefonseca commented:
1. There are 3 places where its was previously passing include-deleted
![image](https://github.com/user-attachments/assets/77071b99-7b78-458d-a2f1-692e096eeb6a)
2. Are you also fixing this edit form?

tahmeedshah commented:
> 1. There are 3 places where its was previously passing include-deleted
>    ![image](https://private-user-images.githubusercontent.com/7049230/413025207-77071b99-7b78-458d-a2f1-692e096eeb6a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzk0NzMzMzEsIm5iZiI6MTczOTQ3MzAzMSwicGF0aCI6Ii83MDQ5MjMwLzQxMzAyNTIwNy03NzA3MWI5OS03Yjc4LTQ1OGQtYTJmMS02OTJlMDk2ZWViNmEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDIxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAyMTNUMTg1NzExWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZTg1Nzg4OGZhYmI1OWMxMDBhYjRiYTg2NzJlYTcxZDdkZjRmODNkNmMwNjEyNThmOTIwMTk3YWI4YzJkYjlhOCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.-IzAo5eqw3e8TUyOH-qKmy59sDHQz1dMKMVudLjcdSg)
> 2. Are you also fixing this edit form?

tahmeedshah commented:
![image](https://github.com/user-attachments/assets/264cf5de-595a-4c1b-b3df-936613736fa6)
in this PR i updated only view page.
Note: for deleted records , it goes the else part, thats why i added inculed-deleted in else part.
